tests/test_inheritance.py

```diff
@@ -143,3 +144,48 @@ class Location(BaseModel, metaclass=ExtendableModelMeta):
 
     schema = Location.schema()
     assert schema is not None
+
+
+def test_gennerics(test_registry):
+    """In this test we chack that the extension also workks with python generics."""
+    T = TypeVar("T")
+
+    class SearchResult(GenericModel, Generic[T], metaclass=ExtendableModelMeta):
+        total: int
+        results: List[T]
+
+    class Location(BaseModel, metaclass=ExtendableModelMeta):
+        kind: Literal["view", "bin"]
+        my_list: List[str]
+
+    class SearchLocationResult(SearchResult[Location]):
```

Comment:
@sbidoul pydantic + generics + extendable .... I still need to check and fix the code when tests run on pypy3 and py<3.8 ....

Reply:
at this stage I dropped the support for generics in for py < 3.8 et pypy3